src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,343 @@
+import * as React from 'react';
+import { If, Then, Else } from 'react-if';
+import * as _ from 'lodash';
+import {observer} from "mobx-react";
+import {
+    IClinicalGroupMatch, IGenomicGroupMatch, IGenomicMatch, ITrial,
+    ITrialMatch, IDetailedTrialMatch, IArmMatch
+} from "../../../shared/model/MatchMiner";
+import styles from './style/trialMatch.module.scss';
+import { computed } from "mobx";
+import LazyMobXTable from "../../../shared/components/lazyMobXTable/LazyMobXTable";
+import SampleManager from "../sampleManager";
+import DefaultTooltip, { placeArrowBottomLeft } from "../../../shared/components/defaultTooltip/DefaultTooltip";
+
+export type ITrialMatchProps = {
+    sampleManager?: SampleManager | null;
+    trials: Array<ITrial>;
+    trialMatches:Array<ITrialMatch>;
+    containerWidth: number;
+}
+
+export type ITrialMatchState = {
+    detailedTrialMatches: Array<IDetailedTrialMatch>;
+}
+
+enum ColumnKey {
+    ID = 'ID',
+    TITLE = 'Title',
+    MATCHINGCRITERIA = 'Matching Criteria'
+}
+
+class TrialMatchTableComponent extends LazyMobXTable<IDetailedTrialMatch> {
+
+}
+
+@observer
+export default class TrialMatchTable extends React.Component<ITrialMatchProps, ITrialMatchState> {
+
+    constructor(props: ITrialMatchProps) {
+        super(props);
+        this.state = { detailedTrialMatches: this.buildDetailedTrialMatches()};
```

Comment:
I don't think `detailedTrialMatches` needs to be listed in the state. this is just one time thing, probably we could just make a variable. And we don't particularly work with state directly since we are using mobx.

Reply:
I moved to  detailedTrialMatches to render().